.github/workflows/build.yaml

```diff
@@ -0,0 +1,88 @@
+name: indy-node-build
+on: [ push, pull_request ]
+
+jobs:
+  decorator-checks:
+    name: Run decorator checks
+    needs: lint
+    runs-on: ubuntu-18.04
+    #TODO: move this to hyperledger?
+    container: m00sey/indy-node-build:latest
```

Comment:
@m00sey, This references will need to be updated.

Reply:
By updated do you mean pushing the image to hyperledger?

Reply:
Ultimately, yes.  However we should discuss how to manage the image since it contains dependencies that will need to be updated occasionally.